.travis.yml

```diff
@@ -150,16 +149,6 @@ install:
           export QT_API=pyqt;
       fi
 
-    # Reinstalling the required numpy version as it may be downgraded.
-    # TODO It may be removed when/if Sherpa becomes np version agnostic
-    # (maybe as soon as v4.8). See discussion in
-    # https://travis-ci.org/gammapy/gammapy/jobs/99867950#L639
-    - if $SHERPA; then
-          conda install --yes --channel astropy numpy=$NUMPY_VERSION iminuit;
-          conda install --yes --no-pin --channel https://conda.anaconda.org/cxc/channel/dev sherpa;
```

Comment:
@cdeil - Build is added for the dev version, let's see whether travis likes it or not. You can also cancel the first two builds on travis, I didn't mean to push them separately.